CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 * Ignore unused import statements which occur in `__init__.py` (RJ722, #192).
+* When reporting properties, use lineno on which the corresponding method is
```

Comment:
"Report first line number of method definition for unused properties (RJ722, #200)."

Reply:
Done.